dockstore-webservice/src/main/java/io/dockstore/webservice/core/Entry.java

```diff
@@ -28,15 +28,18 @@
 import io.swagger.annotations.ApiModelProperty;
 import io.swagger.v3.oas.annotations.media.Schema;
 import java.sql.Timestamp;
+import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.Date;
 import java.util.HashMap;
 import java.util.LinkedHashSet;
+import java.util.List;
 import java.util.Map;
 import java.util.Objects;
 import java.util.Set;
 import java.util.SortedSet;
 import java.util.TreeSet;
+// import java.util.stream.Collectors;
```

Comment:
Avoiding checking in commented code unless there is a code comment saying under what condition it will be uncommented (pretty rare)